src/test/kotlin/com/expediagroup/sdk/core/configuration/provider/FileSystemConfigurationProviderTest.kt

```diff
@@ -50,10 +52,25 @@ internal class FileSystemConfigurationProviderTest {
     }
 
     @Test
-    fun `should return configurations when paths are set and files exist`() {
+    fun `should return null configurations when paths are set and files exist but empty`() {
```

Comment:
Thanks for insisting on the highest standards of testing! keep up the good work!